packages/medusa/src/commands/utils/get-migrations.js

```diff
@@ -104,7 +104,11 @@ export function getInternalModules(configModule) {
 
     let loadedModule = null
     try {
-      loadedModule = require(moduleResolution.moduleDeclaration.resolve).default
+      const resolutionPath =
+        moduleResolution.moduleDeclaration?.resolve ??
+        moduleResolution.resolutionPath
```

Comment:
@carlos-r-l-rodrigues Can I get you to review this change?

In the case of default modules, the `resolve` will not be set, so we will default to the `resolutionPath` as defined in the module definition.

E.g. the local EventBus module:
```
{
    key: "eventBus",
    registrationName: "eventBusModuleService",
    defaultPackage: "@medusajs/event-bus-local",
    label: "EventBusModuleService",
    canOverride: true,
    isRequired: true,
    defaultModuleDeclaration: {
      scope: MODULE_SCOPE.INTERNAL,
      resources: MODULE_RESOURCE_TYPE.SHARED,
    },
  },
```

Reply:
You can use `require(moduleResolution.resolutionPath).default` directly. The `registerModuleDifinitions` will populate that resolutionPath property. Nice catch.